src/extract/constraints.jl

```diff
@@ -91,22 +91,22 @@ function update_ConstraintFunctionsLibrary!()
     ConstraintFunction(:min_required_power_electric_net, "%", dd -> (@ddtime(dd.balance_of_plant.power_electric_net) - dd.requirements.power_electric_net) / dd.requirements.power_electric_net, >, 0.0)
-    ConstraintFunction(:required_flattop, "%", dd -> abs(dd.build.oh.flattop_duration - dd.requirements.flattop_duration) / dd.requirements.flattop_duration, ==, 0.0, 0.01) # relative tolerance
-    ConstraintFunction(:min_required_flattop, "%", dd -> (dd.build.oh.flattop_duration - dd.requirements.flattop_duration) / dd.requirements.flattop_duration, >, 0.0)
+    ConstraintFunction(:required_flattop_duration, "%", dd -> abs(dd.build.oh.flattop_duration - dd.requirements.flattop_duration) / dd.requirements.flattop_duration, ==, 0.0, 0.01) # relative tolerance
+    ConstraintFunction(:min_required_flattop_duration, "%", dd -> (dd.build.oh.flattop_duration - dd.requirements.flattop_duration) / dd.requirements.flattop_duration, >, 0.0)
     ConstraintFunction(:min_required_B0, "%", dd -> (abs(prod(build_max_R0_B0(dd.build))) - dd.equilibrium.vacuum_toroidal_field.r0 * maximum(abs, dd.equilibrium.vacuum_toroidal_field.b0)) / (dd.equilibrium.vacuum_toroidal_field.r0 * maximum(abs, dd.equilibrium.vacuum_toroidal_field.b0)), >, 0.0)
     ConstraintFunction(:zero_ohmic, "MA", dd -> abs(Ip_ohmic(dd.core_profiles.profiles_1d[], dd.equilibrium.time_slice[])) / 1E6, ==, 0.0, 0.1) # absolute tolerance
     ConstraintFunction(:max_ne_peaking, "%", dd -> ((@ddtime(dd.summary.local.magnetic_axis.n_e.value) / @ddtime(dd.summary.volume_average.n_e.value)) - dd.requirements.ne_peaking) / dd.requirements.ne_peaking, <, 0.0)
-    ConstraintFunction(:min_lh_power_threshold, "%", dd -> (L_H_threshold(dd) / dd.requirements.lh_power_threshold_fraction), >, 1.0)
+    ConstraintFunction(:min_lh_power_threshold_fraction, "%", dd -> (L_H_threshold(dd) / dd.requirements.lh_power_threshold_fraction), >, 1.0)
     ConstraintFunction(:max_ωpe_ωce, "%", dd -> ω_pe(@ddtime(dd.summary.local.magnetic_axis.n_e.value)) / ω_ce(@ddtime(dd.equilibrium.vacuum_toroidal_field.b0)), <, 1.0)
     ConstraintFunction(:max_qpol_omp, "%", dd -> (q_pol_omp_eich(dd) - dd.requirements.q_pol_omp) / dd.requirements.q_pol_omp, <, 0.0)
-    ConstraintFunction(:max_tf_j, "%", dd -> (1. + dd.requirements.coil_j_margin) * dd.build.tf.max_j / dd.build.tf.critical_j, <, 1.0)
-    ConstraintFunction(:max_oh_j, "%", dd -> (1. + dd.requirements.coil_j_margin) * dd.build.oh.max_j / dd.build.oh.critical_j, <, 1.0)
+    ConstraintFunction(:max_tf_coil_j_margin, "%", dd -> (1. + dd.requirements.coil_j_margin) * dd.build.tf.max_j / dd.build.tf.critical_j, <, 1.0)
+    ConstraintFunction(:max_oh_coil_j_margin, "%", dd -> (1. + dd.requirements.coil_j_margin) * dd.build.oh.max_j / dd.build.oh.critical_j, <, 1.0)
```

Comment:
These constraints are actually on the maximum tf and oh currents expressed as fraction with respect to the critical currents, including the margins. I think the `max_tf_j` and `max_oh_j` may be more appropriate

Reply:
Got rid of margin on both this one and the stresses in the latest commit